be2-scala/src/test/scala/util/examples/data/builders/HighLevelMessageGenerator.scala

```diff
@@ -153,6 +153,11 @@ object HighLevelMessageGenerator {
           messageData = OpenElection.buildFromJson(payload)
           params = new ParamsWithMessage(paramsChannel, message.withDecodedData(messageData).toMessage)
           JsonRpcRequest(RpcValidator.JSON_RPC_VERSION, methodType, params, id)
+        
+        case (ObjectType.TRANSACTION, ActionType.POST) =>
```

Comment:
Out of curiosity, what is a "TRANSACTION"? Looks like this pull request is for election OPEN but seems to already be implemented above on lines 152-155

Reply:
That code is from Digital Cash. I believe you are looking at the wrong diff from the merge commit.